migrations/20240228-add_column_templateVersionId_to_pipeline.JS

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}pipelines`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.addColumn(
+                table,
+                'templateVersionId',
+                {
+                    type: Sequelize.INTEGER
```

Comment:
To keep it consistent
https://github.com/screwdriver-cd/data-schema/blob/master/migrations/20230727160440-initdb-pipelineTemplateVersions.js#L19
```suggestion
                    type: Sequelize.INTEGER.UNSIGNED
```